docs/source/basic_setup.rst

```diff
@@ -0,0 +1,113 @@
+===========
+Basic Setup
+===========
+
+SubmitCGAP is a Python based tool and requires a Python installation with version >=3.7.
+This document is intended for users who are not very familiar with the command line or Python.
+The intent is to provide detailed instructions for setting up your local environment for using SubmitCGAP.
+These instructions are intended to work with Mac OS X. Linux users are presumed advanced and Windows instructions are not available at this time but will be provided in the future.
+
+-----------------------
+Unix Command Cheatsheet
+-----------------------
+
+Using this repository and interacting with file submissions assumes some knowledge of the Unix
+filesystem and familiarity with bash. What follows is a list of the essential commands you should
+know for working with SubmitCGAP. Whenever in doubt, you can enter "man <cmd>" into the terminal to
+pull up documentation on the command. You can also do such search into your favorite search engine to
+locate the manual pages. Generally you need to know how to look around the file system and create
+directories and files.
+
+* `pwd` will "print working directory" ie: the current location in the Unix file system where the terminal is "located". Whenever you start a new terminal session, odds are you will be sent to your home directory. The `~` character is a global alias for the current active users home directory.
+* `ls` will list all files and directories in the current directory
+
+
+.. code-block:: bash
+
+    $ pwd
+    /Users/<your_username>
+    $ ls
+    directory1
+    file1
+    file2
+    $ ls ~  # this is an alias for your home directory, usually /Users/<your_username>
+    directory1
+    file1
+    file2
+
+
+* `cd` will change directory. Use `ls` first to see which directories are available, then use `cd` to change back to them. The special identifier `..` indicates the directory above the current one, so use `cd ..` to exit
+* `cat <filename>` will output the contents of a file to the terminal
+* `mkdir <directory_name>` will create a new directory
+* `touch <filename>` will create a new empty file
+
+
+Getting Started
+---------------
+
+To get started we will create an empty file to hold the submission credentials and a directory
+for storing the submission Excel files and associated raw sequencing files. At this time, the sequencing
+files need to be on your local machine to be submitted to CGAP. Do this by using the `touch` and `mkdir`
+commands described above. Use the exact command below for creating the credential file, but you can
+create the directory for the submission files anywhere, just note the location (you can check with `pwd`).
+
+
+.. code-block:: bash
+
+    $ touch ~/.cgap-keys.json
+    $ mkdir Documents/submit_cgap
```

Comment:
This code block shows a "$" at the start, while others do not. I don't know for sure if care which, but it would be good to have a consistent theory .

Reply:
Probably I think unless the "$" is adding something important it's easier to cut&paste if those are not there...